pkg/operator/config.go

```diff
@@ -47,11 +48,12 @@ type Config struct {
 }
 
 type ReloaderConfig struct {
-	CPURequest    string
-	CPULimit      string
-	MemoryRequest string
-	MemoryLimit   string
-	Image         string
+	CPURequest      string
+	CPULimit        string
+	MemoryRequest   string
+	MemoryLimit     string
+	Image           string
+	ImagePullPolicy v1.PullPolicy
```

Comment:
The image pull policy should be treated the same way as the other reloader options that depend on the Prometheus/Alertmanager object. E.g. it should have an option function like [LogFormat()](https://github.com/prometheus-operator/prometheus-operator/blob/49a12372e228f881ea388d480b45736fa6cc97cd/pkg/operator/config-reloader.go#L104-L109) that gets called when [generating](https://github.com/prometheus-operator/prometheus-operator/blob/49a12372e228f881ea388d480b45736fa6cc97cd/pkg/prometheus/statefulset.go#L915-L926) the config reloader containers.

Reply:
Actioned.